src/routes/posts.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express')
+
+const db = require('../db')
+
+const router = express.Router()
+
+// CREATE AN ARTICLE
+
+const createPosts = async (req, res) => {
```

Comment:
i think `createPost` will cover it if its just one we're creating? so its easier for a new person to read and understand

Reply:
I have worked on this review now

Reply:
okay, yet to push?

Reply:
![Screenshot from 2022-08-17 12-30-07](https://user-images.githubusercontent.com/68056104/185122305-afb9225d-f74d-4b0c-9a1a-cab1739f95d2.png)
 I have pushed it already, 30 mins ago, please confirm if you see it now

Reply:
@beblicarl yea got it. this change was just about the naming of the function. since its one post being created at a go. just `createPost` without the `s` is fine

Reply:
I have just seen this, i will work on it as well